cardinal/server/server_test.go

```diff
@@ -134,50 +134,50 @@ func TestHandleTransactionWithNoSignatureVerification(t *testing.T) {
 	assert.Equal(t, 1, count)
 }
 
-func TestHandleWrappedTransactionWithNoSignatureVerification(t *testing.T) {
```

Comment:
does this still need to be commented out?

Reply:
in any case, you can just put a `t.Skip()` at the begging of the test, and maybe add a comment as to why we are skipping, whats broken, etc

Reply:
the test works on my end btw 🤔

Reply:
ahh wait a second.. i think the test needs to use httpmock instead of the actual http package to work in CI...

Reply:
yeah I'm not sure either bcs the other http test is working, only in this case it's failing.

I'll check it out a bit more, or maybe will add `t.Skip()` for now.